README.md

```diff
@@ -73,14 +73,17 @@ Prior to v2.0, `mfa_method` is always required.
 #### Option 1: TOTP
 Set `mfa_method` to `soft-token`.
 
-Set `mfa_token` as follows:
+Obtain the seed for `mfa_token` as follows:
 go to [your Mint settings](https://mint.intuit.com/settings.event?filter=all),
 navigate through *Intuit Account* -> *Sign In & Security* -> *Two-step verification*.
-From there, enable either text or email as desired.
-After that, start the process to enable the *Authenticator app* option and when you get to the part where you see the QR code,
-**copy the manual setup code** that appears next to it. 
-**BE CAREFUL WHERE YOU STORE THIS**, as anyone with it will be able to take over your Mint account.
+Token MFA is only allowed as a secondary option,
+so first enable one of the other options (text, email, voice).
+Select the *Authenticator app* option.
+When you get to the part where you see the QR code,
+copy the **manual setup code** that appears next to it.
 This is the token you pass to `mfa_token` in either the python api or from the command line.
+Complete the setup process by providing a generated code from your authenticator app when requested.
+**BE CAREFUL WHERE YOU STORE THIS**, as anyone with it will be able to take over your Mint account.
```

Comment:
While here, can you convert this markdown to a list?

Reply:
Converted to list.